sdk/metric/provider_test.go

```diff
@@ -74,3 +88,17 @@ func TestMeterProviderReturnsSameMeter(t *testing.T) {
 	assert.Same(t, mtr, mp.Meter(""))
 	assert.NotSame(t, mtr, mp.Meter("diff"))
 }
+
+func TestMeterProviderReturnsNoopMeterAfterShutdown(t *testing.T) {
+	mp := MeterProvider{}
```

Comment:
Just to ensure a normal functioning setup:

```suggestion
	mp := NewMeterProvider()
```